Modules/Filtering/ImageIntensity/include/itkMaskImageFilter.h

```diff
@@ -94,22 +91,21 @@ class MaskInput
 
 private:
   template <typename TPixelType>
-  void
-  InitializeOutsideValue(TPixelType *)
+  TPixelType
+  DefaultOutsideValue(TPixelType *)
   {
-    this->m_OutsideValue = NumericTraits<TPixelType>::ZeroValue();
+    return this->m_OutsideValue = NumericTraits<TPixelType>::ZeroValue();
```

Comment:
Shouldn't this be two lines?

Reply:
Good eye, that assignment should not be there.